mne/annotations.py

```diff
@@ -358,9 +369,24 @@ def _combine_annotations(one, two, one_n_samples, one_first_samp,
 
 
 def _handle_meas_date(meas_date):
-    """Convert meas_date to seconds."""
+    """Convert meas_date to seconds.
+
+    if meas_date is string, it should conform to '%Y-%m-%d %H:%M:%S.%f'
```

Comment:
Captitalize "If". There are a couple of typos, here's my suggestion:

```
    """Convert meas_date to seconds.

    If `meas_date` is a string, it should conform to the ISO8601 format, i.e.
    '%Y-%m-%d %H:%M:%S.%f'. Otherwise, this function returns 0. Note that
    ISO8601 allows for ' ' or 'T' as delimiters between date and time.
    """

However, I'm not sure if you meant to write `'\t'` instead of `'T'` in the last sentence - could you please check?

Reply:
I see that the regex does have `[ T]` so `T` is OK.

Reply:
yes the regex conforms to the standard, which is with ' ' or 'T'. But until now I did only implemted the ' '. Thats why the test says invalid. It should be straight forward to implement if needed but for now YAGNI.

Reply:
Then you should clarify in the comment that 'T' is not supported ATM.